src/sentry/grouping/enhancer/enhancement-configs/mobile@2021-04-02.txt

```diff
@@ -80,8 +80,8 @@ error.mechanism:ANR invert-stacktrace=1
 error.mechanism:NSError invert-stacktrace=1
 
 # Dart/Flutter stacktraces that are not in-app
-family:javascript stack.abs_path:org-dartlang-sdk* -app -group
-family:javascript module:flutter/flutter/packages/flutter/* -app
+family:javascript stack.abs_path:org-dartlang-sdk:///** -app -group
+family:javascript module:**/packages/flutter/** -app
```

Comment:
@untitaker should we also start with `**` since it could be many folders?

eg: opt/hostedtoolcache/flutter/2.5.0-stable/x64/`packages/flutter`/lib/src/gestures/binding

Reply:
Do we have a test for that? I don't see snapshot changes as a result of the change on Line 84, only for Line 83

Reply:
good catch, just added `frame-ignores-flutter-sdk.json`, could you update the snapshot tests please since I don't have the sentry env on my machine, that would be very helpful, thanks